setup.py

```diff
@@ -47,6 +47,7 @@
         "uvicorn",
         "Jinja2",
         "fsspec",
+        "h11==0.12.0",
```

Comment:
@aliabid94 I can't repro the original problem on main. I did the following:

```bash
conda create -n test python=3.9 pip -y
conda activate test
python setup.py sdist
pip install dist/gradio-3.0.22.tar.gz
```

And the install succeeded. How can I reproduce?

Reply:
Here it is in a google colab. 
https://colab.research.google.com/drive/1sRcgYQoJ5IOaHRO-QRm2-26CWIcg1qLb?usp=sharing
The error might be flaky depending on the order that the dependencies are installed.

Reply:
Thank you for the link @aliabid94 !